docs/en/ingest-management/release-notes/release-notes-7.14.asciidoc

```diff
@@ -0,0 +1,138 @@
+// Use these for links to issue and pulls. 
+:kib-issue: https://github.com/elastic/kibana/issues/
+:kib-pull: https://github.com/elastic/kibana/pull/
+:agent-issue: https://github.com/elastic/beats/issues/
+:agent-pull: https://github.com/elastic/beats/pull/
+:fleet-server-issue: https://github.com/elastic/beats/issues/fleet-server/
+:fleet-server-pull: https://github.com/elastic/beats/pull/fleet-server/
+
+[[release-notes]]
+= Release notes
+
+This section summarizes the changes in each release.
+
+* <<release-notes-7.14.0>>
+
+Also see:
+
+* {kibana-ref}/release-notes.html[{kib} release notes]
+* {beats-ref}/release-notes.html[{beats} release notes]
+
+[[release-notes-7.14.0]]
+== {fleet} and {agent} 7.14.0
+
+Review important information about the {fleet} and {agent} 7.14.x releases.
+
+//[discrete]
+//[[security-updates-7.14.0]]
+//=== Security updates
+
+//{fleet}::
+//* add info
+
+//{agent}::
+//* add info
+
+//[discrete]
+//[[breaking-changes-7.14.0]]
+//=== Breaking changes
+
+//Breaking changes can prevent your application from optimal operation and
+//performance. Before you upgrade, review the breaking changes, then mitigate the
+//impact to your application.
+
+//[discrete]
+//[[breaking-PR#]]
+//.Short description
+//[%collapsible]
+//====
+//*Details* +
+//<Describe new behavior.> For more information, refer to {kibana-pull}PR[#PR].
+
+//*Impact* +
+//<Describe how users should mitigate the change.> For more information, refer to {fleet-guide}/fleet-server.html[Fleet Server].
+//====
+
+[discrete]
+[[known-issues-7.14.0]]
+=== Known issues
+
+[[known-issue-1]]
+.Credentials potentially exposed in `.fleet-*` and `.kibana*` indices
+[%collapsible]
+====
+
+*Details* 
+
+// tag::credential-storage[]
+
+Credentials that you provide for an agent or integration policy are stored in
+{es}. They can be read by any user who has read permissions to the `.fleet-*`
+and `.kibana*` indices in {es}. By default these are the superuser,
+`fleet-server` service account tokens, and the `kibana_system` user. These
+secrets are also included in agent polices and shared with agents via {fleet}
+through TLS. When you use the {agent} installer and enroll agents in {fleet},
+the policies are stored on the host file system and, by default, can only be
+read by root.
+
+// end::credential-storage[]
+
+*Impact* +
+
+To avoid exposing secrets, restrict access to the `.fleet-*` and `.kibana*`
+indices to trusted users. Do not change file ownership or permissions on policy
+files stored on the host file system unless you can secure the files from
+malicious users.
+
+//Jason: ^^ Example of what you might want to say in the impace section.
+//Also, is there a tracking issue we can refer to here? We usually add a link to
+//related GitHub issue.
```

Comment:
there is this one but it is private: https://github.com/elastic/obs-dc-team/issues/535

this one is related: https://github.com/elastic/integrations/issues/244

@mostlyjason do you know of any other issues?

Reply:
No and this is a little different than other release notes. This not something that's changing in this release, we are just documenting a limitation of the release that has always existed. Its more like a standalone FYI or a comment. If this is not appropriate for the release notes just let us know. We also document it in the FAQ.

Reply:
Let's leave it in the release notes for now. It might make more sense to mention this in the docs we add for https://github.com/elastic/observability-docs/issues/605